### ISSUE TITLE
remmina not compiling on cent os 6.4 x64

### ISSUE THREAD

**Issue by beezel** [FreeRDP/Remmina#201]
During 'sudo make install' I recieve the following errors:

Scanning dependencies of target remmina
[  0%] Building C object remmina/CMakeFiles/remmina.dir/src/remmina_about.c.o
[  1%] Building C object remmina/CMakeFiles/remmina.dir/src/remmina_applet_menu.c.o
[  2%] Building C object remmina/CMakeFiles/remmina.dir/src/remmina_applet_menu_item.c.o
[  3%] Building C object remmina/CMakeFiles/remmina.dir/src/remmina_avahi.c.o
[  4%] Building C object remmina/CMakeFiles/remmina.dir/src/remmina.c.o
/home/beezel/source/FreeRDP-Remmina-356c033/remmina/src/remmina.c:73: error: expected ‘)’ before ‘*’ token
/home/beezel/source/FreeRDP-Remmina-356c033/remmina/src/remmina.c:170: error: expected ‘)’ before ‘*’ token
/home/beezel/source/FreeRDP-Remmina-356c033/remmina/src/remmina.c: In function ‘main’:
/home/beezel/source/FreeRDP-Remmina-356c033/remmina/src/remmina.c:189: error: ‘GApplication’ undeclared (first use in this function)
/home/beezel/source/FreeRDP-Remmina-356c033/remmina/src/remmina.c:189: error: (Each undeclared identifier is reported only once
/home/beezel/source/FreeRDP-Remmina-356c033/remmina/src/remmina.c:189: error: for each function it appears in.)
/home/beezel/source/FreeRDP-Remmina-356c033/remmina/src/remmina.c:189: error: ‘app’ undeclared (first use in this function)
/home/beezel/source/FreeRDP-Remmina-356c033/remmina/src/remmina.c:210: warning: implicit declaration of function ‘g_application_new’
/home/beezel/source/FreeRDP-Remmina-356c033/remmina/src/remmina.c:210: error: ‘G_APPLICATION_HANDLES_COMMAND_LINE’ undeclared (first use in this function)
/home/beezel/source/FreeRDP-Remmina-356c033/remmina/src/remmina.c:211: error: ‘remmina_on_startup’ undeclared (first use in this function)
/home/beezel/source/FreeRDP-Remmina-356c033/remmina/src/remmina.c:212: error: ‘remmina_on_command_line’ undeclared (first use in this function)
/home/beezel/source/FreeRDP-Remmina-356c033/remmina/src/remmina.c:213: warning: implicit declaration of function ‘g_application_set_inactivity_timeout’
/home/beezel/source/FreeRDP-Remmina-356c033/remmina/src/remmina.c:215: warning: implicit declaration of function ‘g_application_run’
/home/beezel/source/FreeRDP-Remmina-356c033/remmina/src/remmina.c:217: warning: implicit declaration of function ‘g_application_get_is_remote’
At top level:
cc1: warning: unrecognized command line option "-Wno-unused-result"
make[2]: *** [remmina/CMakeFiles/remmina.dir/src/remmina.c.o] Error 1
make[1]: *** [remmina/CMakeFiles/remmina.dir/all] Error 2
make: *** [all] Error 2

**Comment by Hubbitus** [comment]
Does it actual at the moment?

**Comment by giox069** [comment]
This issue is too old and should be closed because it refers to an old and not maintained verson of remmina.
Remmina 1.2 only compiles with GTK version 3.X.
If you need a GTK2 compatible version, try with the branch "gtk2" of this repository, and reopen a new issue for problems with it. But be aware that the "gtk2" branch is not very well supported.
